scripts/gha/print_matrix_configuration.py

```diff
@@ -169,8 +169,8 @@
   "emulator_target": {"type": "virtual", "image":"system-images;android-28;google_apis;x86_64"},
   "emulator_latest": {"type": "virtual", "image":"system-images;android-30;google_apis;x86_64"},
   "emulator_32bit": {"type": "virtual", "image":"system-images;android-30;google_apis;x86"},
-  "ios_min": {"type": "real", "model":"iphone8", "version":"11.4"},
-  "ios_target": {"type": "real", "model":"iphone8plus", "version":"12.0"},
+  "ios_min": {"type": "real", "model":"iphone8", "version":"12.4"},
```

Comment:
12.4 is with TAG `deprecated`:
`gcloud firebase test ios models list`

I would suggest update all iOS devices to: 13.2, 14.7, 15.4
I also checked android devices, looks good for now.